examples/app_placement/app_v2.py

```diff
@@ -0,0 +1,52 @@
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from uvicorn import run
+
+from lightning import CloudCompute, LightningApp, LightningFlow, LightningWork
+
+
+class Work(LightningWork):
+    def __init__(self, **kwargs):
+        super().__init__(parallel=True, **kwargs)
+
+    def run(self):
+        fastapi_service = FastAPI()
+
+        fastapi_service.add_middleware(
+            CORSMiddleware,
+            allow_origins=["*"],
+            allow_credentials=True,
+            allow_methods=["*"],
+            allow_headers=["*"],
+        )
+
+        @fastapi_service.get("/")
+        def get_root():
+            return {"Hello Word!"}
+
+        run(fastapi_service, host=self.host, port=self.port)
+
+
+class Flow(LightningFlow):
+    def __init__(self):
+        super().__init__()
+        self.cloud_compute = CloudCompute(name="cpu-small")
+        self.work_a = Work()
```

Comment:
Since this is an example, I would add comments to make sure readers know what they’re looking at (namely placement, since it’s not very explicit especially with default work)

Reply:
Done !